### DISCUSSION TITLE
Loki/Prometheus query builder: Improve drag and drop

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#52096] [Observability]
It is not working the best when you have many operations and they wrap. This is not an easy fix, no drag and drop library we have found supports this for elements of dynamic width.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
